kuma/users/views.py

```diff
@@ -179,61 +181,91 @@ def ban_user_and_cleanup_summary(request, user_id):
             submitted_to_akismet.append(revision)
         else:
             not_submitted_to_akismet.append(revision)
+
+        # If there is a current revision and the revision is not in the spam list,
+        # to be reverted, do not revert any revisions
+        try:
+            revision.document.refresh_from_db(fields=['current_revision'])
+        except Document.DoesNotExist:
+            continue  # This document was previously deleted in this loop, continue
+        if revision.document.current_revision not in revisions_to_mark_as_spam_and_revert:
+            if revision.document_id not in previous_good_rev:
+                previous_good_rev[revision.document_id] = revision.document.current_revision
+
+            continue  # This document has a more current revision, no need to revert
```

Comment:
This line is [not covered](https://codecov.io/gh/mozilla/kuma/src/1203d19f6d3fe1b66e609a1e62b74525c7816744/kuma/users/views.py#L195), which confuses me greatly. This may be a bug in coverage, but I'll think more about it.